### ISSUE TITLE
distributions include both Makefile.PL and Build.PL

### ISSUE THREAD

**Issue by karenetheridge** [ThePilgrim/perlcldr#6]
Providing both files is generally not a good idea -- e.g. see http://neilb.org/2015/05/18/two-build-files-considered-harmful.html

Module::Build::Compat especially is a giant bag of razorblades and best avoided.  The easiest thing to do is to drop the use of the 'compat' option in Module::Build, but since you're providing a Makefile.PL with no evident loss of functionality, even better would be to drop the use of Module::Build entirely and just use ExtUtils::MakeMaker in Makefile.PL.

**Comment by ThePilgrim** [comment]
Thanks for the tip.
I was cargo culting the build process when I first published to CPAN and haven't really looked at it  since then.

I have removed the create_makefile_pl line from the Build,PL script. This should not cause a problem as the code does not run on anything older than Perl 5.10.0 and my, somewhat limited, research leads me to believe that the CPAN tool chain that came with 5.10.0 understands the configure requires meta data so Module::Build will be installed before my distribution.

I have also spotted other problems with the distributions I produce that I will fix for the next release.

Thanks again
John
